src/main/kotlin/no/nav/familie/ef/sak/service/BehandlingService.kt

```diff
@@ -71,6 +71,15 @@ class BehandlingService(private val behandlingsjournalpostRepository: Behandling
                                                       resultat = BehandlingResultat.IKKE_SATT))
     }
 
+    fun opprettBehandling(behandlingType: BehandlingType, fagsakId: UUID, status: BehandlingStatus): Behandling {
```

Comment:
Går det å utvide den andre `opprettBehandling` med `status: BehandlingStatus.OPPRETTET`? 

Samtidig er jeg litt usikker på om det er riktig å bruke `steg = StegType.VILKÅR`, i dette tilfelle burde den vel være `VENTE_PÅ_STATUS_FRA_IVERKSETT` ? 
Sånn det er nå kommer vel denne hoppe fra `VILKÅR` til `FERDIGSTILLE_BEHANDLING`? 

Litt intressant att BA har dette flytet for teknisk opphør. Til senere: Burde vi ha et eget steg for Teknisk opphør? Jeg tror ikke vi trenger totrinnskontroll av teknisk opphør? Burde vi bare checke det med funksjonelle? 
```
REGISTRERE_PERSONGRUNNLAG -> VILKÅRSVURDERING
VILKÅRSVURDERING -> SEND_TIL_BESLUTTER
SEND_TIL_BESLUTTER -> BESLUTTE_VEDTAK
BESLUTTE_VEDTAK -> IVERKSETT_MOT_OPPDRAG
```